src/features/session_trace/aggregate/index.js

```diff
@@ -201,10 +201,10 @@ export class Aggregate extends AggregateBase {
   }
 
   /** Stop running for the remainder of the page lifecycle */
-  abort (reason) {
+  abort () {
```

Comment:
We should probably have a separate ticket to add "abort reason" SMs here.  They have been very useful for SR and I suspect would have some utility with ST as well

Reply:
I can revert the removal of the reason codes if you want. They were actually very helpful in determining where in the code the call to abort was coming from.

Reply:
Yeah that's reasonable for now